src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -1169,7 +1169,15 @@ private bool IsValidOverrideReturnType(Symbol overridingSymbol, TypeWithAnnotati
                 location,
                 new FormattedSymbol(overridingParameter, SymbolDisplayFormat.ShortFormat));
 
-        internal static void CheckValidNullableMethodOverride<TArg>(
+        /// <returns>
+        /// Returns true if there is:
+        /// <list type="bullet">
+        /// <item><description>DoesNotReturn mismatch.</description></item>
+        /// <item><description>Parameter type mismatch.</description></item>
+        /// <item><description>Return type mismatch.</description></item>
+        /// </list>
```

Comment:
Consider simplifying the doc comment a la the following:

```suggestion
        /// <see langword="true"/> if a diagnostic was added. Otherwise, <see langword="false"/>.
```